irohad/consensus/yac/transport/impl/network_impl.cpp

```diff
@@ -23,8 +23,10 @@ namespace iroha {
 
```

Comment:
`#include "network/impl/grpc_channel_builder.hpp"` can be removed from this file.